roles/openshift_master/templates/native-cluster/atomic-openshift-master-controllers.service.j2

```diff
@@ -11,6 +11,21 @@ Type=notify
 EnvironmentFile=/etc/sysconfig/{{ openshift_service_type }}-master-controllers
 Environment=GOTRACEBACK=crash
 ExecStart=/usr/bin/openshift start master controllers --config=${CONFIG_FILE} $OPTIONS
+ExecStart=/usr/bin/hyperkube kube-controller-manager \
+  --controllers="*" --controllers=-ttl --controllers=-bootstrapsigner --controllers=-tokencleaner --controllers=-horizontalpodautoscaling --controllers=-serviceaccount-token \
+  --service-account-private-key-file=openshift.local.config/master/serviceaccounts.private.key \
+  --root-ca-file=openshift.local.config/master/ca-bundle.crt \
+  --kubeconfig=openshift.local.config/master/openshift-master.kubeconfig \
+  --pod-eviction-timeout=5m \
+  --enable-dynamic-provisioning=true \
+  --port=-1 \
+  --use-service-account-credentials=true \
+  --cluster-signing-cert-file="" \
+  --cluster-signing-key-file="" \
+  --leader-elect \
+  --leader-elect-retry-period=3s \
+  --leader-elect-resource-lock=configmaps \
+  --openshift-config=${CONFIG_FILE}
```

Comment:
Interesting - I would have thought the other settings would come from openshift-config as well.  Is that not the case?

Reply:
> Interesting - I would have thought the other settings would come from openshift-config as well. Is that not the case?

This file is needed for 
 1. recycler template which should be passed as a fully formed template file reference (flag available)
 2. command line flag args which should be passed as options

I added comments here: https://github.com/openshift/origin/pull/18100/files#r161594020

I think we should try to follow kube when kube moves to its config, not double down on our own for those two settings.  Then its a feature flag and we're only two patches away from running just like upstream (for good or for ill)